modules/placement-driver/src/test/java/org/apache/ignite/internal/placementdriver/leases/LeaseSerializationTest.java

```diff
@@ -65,7 +43,7 @@ public void testLeaseBatchSerialization() {
                     i % 2 == 0,
                     i % 2 == 1,
                     i % 2 == 0 ? null : "node" + i,
-                    groupId
+                    new TablePartitionId(1, i)
```

Comment:
could you pls add tests with different table ids and holes in partition list?

Reply:
The tests you request already exist in `LeaseBatchSerializerTest`, what's the point in duplicating them here?